frontend/src/App.jsx

```diff
@@ -0,0 +1,51 @@
+// App.js
+import { BrowserRouter, Routes, Route, Navigate } from 'react-router-dom';
+import { useState, useEffect } from 'react';
+import Home from './components/Home';
+import About from './components/About';
+import BookList from './components/BookList';
+import NewBook from './components/NewBook';
+import Signup from './components/Signup';
+import Login from './components/Login';
+import Header from './components/Header';
+import CurrentBook from './components/CurrentBook';
+
+const App = () => {
+  const [isUserLoggedIn, setUserLoggedIn] = useState(false);
+  const [currentBook, setCurrentBook] = useState(null);
+
+  useEffect(() => {
+    const isLoggedIn = localStorage.getItem('isLoggedIn') === 'true';
```

Comment:
@omkogde-vf Figure out a way to solve this, "**if we set "isLoggedIn"  to true in localStorage, We get automatically logged in**"